### PR TITLE
Show create table synopsis

### PR DESCRIPTION
**Pull request by mikethebeer**

### CODE REVIEW

File: docs/sql/reference/show_create_table.txt [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+========
+
+::
+  
+    SHOW CREATE TABLE table_ident
+
+Description
+===========
+
+``SHOW CREATE TABLE`` can be used to dump the schema of an existing table.
```
Comment (chaudum) [inline docs/sql/reference/show_create_table.txt]: ... existing **user-created** table.

File: docs/sql/reference/show_create_table.txt [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+Let's create a simple table ``my_table`` with three columns of type ``integer``, ``string`` and ``datetime``::
+
+    cr> create table my_table (
+    ...   first_column integer,
+    ...   second_column string,
+    ...   third_column datetime,
+    ... );
+    CREATE OK (... sec)
+
+Now we print the table DDL syntax using the ``SHOW CREATE TABLE`` statement::
```
Comment (dobe) [inline docs/sql/reference/show_create_table.txt]: put examples somewhere else

File: docs/sql/ddl.txt [hunk 3]
```diff
@@ -812,3 +812,29 @@ node startup defined configuration file value or to its default value::
     +--------------------------------------------------------------------------+
     SELECT 1 row in set (... sec)
 
+
+Show Create Table
+=================
+
+To print a user-created table the ``SHOW CREATE TABLE`` statement can be used. This statement displays the DDL-schema of an existing table in the cluster. The following statement creates a table with ident ``my_table`` and defines it's schema. ::
```
Comment (chaudum) [inline docs/sql/ddl.txt]: leave out 2nd sentence

File: docs/sql/ddl.txt [hunk 4]
```diff
@@ -812,3 +812,29 @@ node startup defined configuration file value or to its default value::
...
+
+    cr> create table my_table (
```
Comment (chaudum) [inline docs/sql/ddl.txt]: hide statement with

`.. hide:`

Comment (chaudum) [inline docs/sql/ddl.txt]: but make it a bit more complex (column constraint, routing column, table settings)

File: docs/sql/ddl.txt [hunk 5]
```diff
@@ -812,3 +812,29 @@ node startup defined configuration file value or to its default value::
...
+    ...   first_column integer,
+    ...   second_column string,
+    ...   third_column datetime,
+    ... );
+    CREATE OK (... sec)
+
+If the ``SHOW CREATE TABLE`` statement gets invoked, the schema of the previous created table gets returned. ::
+
+    cr> show create table my_table;
```
Comment (chaudum) [inline docs/sql/ddl.txt]: please take a look at the `SqlFormatter` to see how the statement will be formatted

File: docs/sql/ddl.txt [hunk 6]
```diff
@@ -812,3 +812,30 @@ node startup defined configuration file value or to its default value::
...
+    +---------------------------------------------------+
+    | show create table my_table                        |
+    +---------------------------------------------------+
+    | CREATE TABLE IF NOT EXISTS "my_table" (           |
+    | ... "first_column" INTEGER PRIMARY KEY,           |
+    | ... "second_column" STRING,                       |
+    | ... "third_column" TIMESTAMP                      |
+    | ... ) CLUSTERED BY ("first_column") INTO 5 SHARDS |
+    +---------------------------------------------------+
```
Comment (chaudum) [inline docs/sql/ddl.txt]: no dots `...`

```
+-----------------------------------------------+
| show create table my_table                    |
+-----------------------------------------------+
| CREATE TABLE IF NOT EXISTS "my_table" (       |
|     "first_column" INTEGER PRIMARY KEY,       |
|     "second_column" STRING,                   |
|     "third_column" TIMESTAMP                  |
| ) CLUSTERED BY ("first_column") INTO 5 SHARDS |
+-----------------------------------------------+
```

**Comment by chaudum** [comment]
otherwise ack

**Pull request merged**